pom.xml

```diff
@@ -79,7 +79,7 @@
         <version.maven-compiler-plugin>3.8.1</version.maven-compiler-plugin>
         <version.maven-jar-plugin>3.2.0</version.maven-jar-plugin>
         <version.maven-surefire-plugin>2.22.2</version.maven-surefire-plugin>
-        <version.quarkus>1.11.1.Final</version.quarkus>
+        <version.quarkus>1.12.0.Final</version.quarkus>
```

Comment:
Please use `1.12.1`

Reply:
We can't use 1.12.1 version because of the issue with OpenShift.